Assets/Scripts/Interface/Options/Options.cs

```diff
@@ -105,6 +106,12 @@ protected void OnEnable()
             SetOptions();
         }
 
+        private void OnFullscreenChange(bool a_fullscreen)
+		{
```

Comment:
I don't know if it's just GitHub (sometimes it just doesn't want to show stuff as intended) or if there's wrong indentation in the original file, double check that!